### PR TITLE
Refactor colorize function away from the test script

### PR DESCRIPTION
**Pull request by alexherbo2**

### CODE REVIEW

**Comment by mawww** [comment]
Does not work here, it seems the change relies on `echo` expanding escape sequences, which is not portable, those echo calls should be replaced with printfs.

**Comment by alexherbo2** [comment]
Done

File: test/run [hunk 1]
```diff
@@ -50,7 +50,7 @@ main() {
       echo "$indent$name"
       continue
     elif [ -x enabled ] && ! ./enabled; then
-      echo "$indent$name (disabled)" | colorize $yellow $none
+      printf "${yellow}$indent$name (disabled)${none}\n"
```
Comment (mawww) [inline test/run]: `$name` could be pretty arbitrary as it comes from the filesystem, so we should probably not put it in the `printf` format string.

**Comment by alexherbo2** [comment]
Done

**Pull request merged**